composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerScreen.kt

```diff
@@ -83,15 +138,15 @@ fun AddButton(clickAdd: () -> Unit) {
 }
 
 @Composable
-fun Workouts(list: List<String>, onDelete: (String) -> Unit) {
+fun Workouts(list: List<String>, onDelete: (Int) -> Unit) {
```

Comment:
```suggestion
fun Workouts(list: List<String>, onDelete: (index: Int) -> Unit) {
```

You can specify parameter names of callbacks to make them even more readable :)